src/shared/utils/text.ts

```diff
@@ -23,7 +23,7 @@ export function parseStreamId(streamId: string): {
     pathname: string
 } {
     try {
-        const [, owner, pathname] = streamId.match(/^(0x[a-f\d]{40})([^\s]+)$/)!
+        const [, owner, pathname] = streamId.match(/^([^/]+)(\/[^\s]+)$/)!
```

Comment:
Guess it's fine to not be so strict about it as long as we drive it. Precise regexp for this would be a bit hairy and I just don't see the need.